src/rt/sched/cown.h

```diff
@@ -977,10 +1105,13 @@ namespace verona::rt
         }
       }
 
+      assert(!backpressure.load(std::memory_order_acquire).muted());
+
       // Now we may run our destructor.
       destructor();
 
       MultiMessage* last = queue.destroy();
+      assert(last->next.load(std::memory_order_relaxed) == nullptr);
```

Comment:
I feel like I sort of know why this is happening based on reading earlier code, but an explanatory comment here would be helpful for me. It feels like a bit of an odd assert when you run across it.

Reply:
This is to check that the message queue is empty when it is destroyed. I found this assert useful to debug premature collection of cowns without needing to hit the queue invariant assert whenever a collected cown is run.